python/py-schema/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-schema
+version             0.7.7
+revision            0
+
+license             MIT
+maintainers         nomaintainer
+
```

Comment:
this port does not appear to install architecture dependent files, so please add:

platforms {darwin any}
supported_archs noarch